src/main/kotlin/uk/gov/gdx/datashare/repository/EventApiAudit.kt

```diff
@@ -0,0 +1,79 @@
+package uk.gov.gdx.datashare.repository
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.databind.ObjectMapper
+import io.swagger.v3.oas.annotations.media.Schema
+import org.postgresql.util.PGobject
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.core.convert.converter.Converter
+import org.springframework.data.annotation.Id
+import org.springframework.data.annotation.Transient
+import org.springframework.data.convert.ReadingConverter
+import org.springframework.data.convert.WritingConverter
+import org.springframework.data.domain.Persistable
+import org.springframework.data.relational.core.mapping.Column
+import java.time.LocalDateTime
+import java.util.UUID
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+class EventApiAudit(
+  @Id
+  @Column("id")
+  @Schema(description = "Audit ID", required = true, example = "00000000-0000-0001-0000-000000000000")
+  val auditId: UUID = UUID.randomUUID(),
+
+  @Schema(description = "Oauth ID of client making call", required = true, example = "alskd987")
+  val oauthClientId: String,
+  @Schema(description = "URL", required = true, example = "https://d33v84mi0vopmk.cloudfront.net/events")
+  val url: String,
+  @Schema(description = "METHOD", required = true, example = "GET")
+  val requestMethod: String,
+  @Schema(description = "Data of event(s)", required = true, example = "{\"data\":[]}")
+  val payload: Payload,
+
+  @Schema(description = "Time of call", required = true)
+  val whenCreated: LocalDateTime,
```

Comment:
Is this actually time of call, or is this when we save it?

Reply:
Its when we save, will update the description